lib/features/overview/bloc/givt_bloc.dart

```diff
@@ -225,4 +227,28 @@ class GivtBloc extends Bloc<GivtEvent, GivtState> {
     }
     return giftAidGroups;
   }
+
+  FutureOr<void> _onGivtDownloadOverviewByYear(
+    GivtDownloadOverviewByYear event,
+    Emitter<GivtState> emit,
+  ) async {
+    emit(const GivtLoading());
+    try {
+      await givtRepository.downloadYearlyOverview(
+        event.guid,
+        event.year,
+      );
+
+      emit(const GivtDownloadedSuccess());
+      add(const GivtInit());
```

Comment:
Why is this needed?

Reply:
Emiting GivtLoading & GivtDownload success erased the givts from state, so i needed to refetch. 
I changed it so GivtDownloadedSuccess carries the givts from prev state.